hubble/ui.go

```diff
@@ -188,7 +188,7 @@ func (k *K8sHubble) generateHubbleUIDeployment() *appsv1.Deployment {
 					Containers: []corev1.Container{
 						{
 							Name:            "frontend",
-							Image:           "quay.io/cilium/hubble-ui:v0.8.3@sha256:018ed122968de658d8874e2982fa6b3a8ae64b43d2356c05f977004176a89310",
```

Comment:
Not blocking this PR, but maybe as a follow-up: It would be nice to keep pinning the default image SHA256 sums. We should probably also add them for all other default images which don't currently have them.

Reply:
Created #648